lib/src/replay/element_parsers/element_data.dart

```diff
@@ -35,4 +41,18 @@ class ElementData {
     }
     return rects;
   }
+
+  void _collectNoMaskWidgetRects(ElementData element, List<Rect> rectList) {
+    if (!rectList.contains(element.rect)) {
+      if (element.type == "PostHogNoMaskWidget") {
+        rectList.add(element.rect);
+      }
+    }
+
+    if (element.children != null && element.children!.isNotEmpty) {
```

Comment:
create a final var with `element.children` so you don't have to use the `!` operator that usually throws errors